packages/core/src/components/expandable/expandable.scss

```diff
@@ -3,10 +3,15 @@
 :host([enable-animation]) {
   display: grid;
   grid-template-rows: 0fr;
+  overflow: hidden;
   @include utilities.transition(grid-template-rows);
+}
 
+:host([enable-animation]:not([open="true"])) {
```

Comment:
[open="true"] is helaas echt nodig. [open] volstaat niet, dan krijg ik geen animatie op het inklappen te zien

Reply:
helaas